packages/deploy/hardhat.config.ts

```diff
@@ -12,7 +12,8 @@ import './tasks/importedPackages';
 
 // Package name : solidity source code path
 const importedPackages = {
-  '@sandbox-smart-contracts/asset': 'contracts/',
+  'asset-latest': 'contracts/',
+  'asset-1.0.2': 'contracts/',
```

Comment:
We can be more specific by using a list that contains only the contracts we need to avoid compiling mocks and other stuff.

Reply:
Possibly, but I see little to no harm if we include all of them.

If we were to do this I'd suggest to move the mocks out of the main `contracts` folder.